bedrock/settings/base.py

```diff
@@ -24,29 +26,31 @@ def path(*args):
 
 
 # Is this a dev instance?
-DEV = False
+DEV = config('DEV', cast=bool, default=False)
 
-DEBUG = False
-TEMPLATE_DEBUG = DEBUG
+DEBUG = TEMPLATE_DEBUG = config('DEBUG', cast=bool, default=False)
 
 # Production uses MySQL, but Sqlite should be sufficient for local development.
 # Our CI server tests against MySQL.
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'bedrock.db',
-    }
-}
-SLAVE_DATABASES = []
+DATABASES = config(
+    'DATABASES',
+    cast=lambda db_json: {
+        db_name: dj_database_url.parse(db_url)
+        for db_name, db_url in json.loads(db_json).items()},
+    default=json.dumps(
+        {'default': config('DATABASE_URL',
+                           default='sqlite:///bedrock.db')}))
+
+SLAVE_DATABASES = config('SLAVE_DATABASES', cast=Csv(), default=',')
 DATABASE_ROUTERS = ('multidb.PinningMasterSlaveRouter',)
 
-# Override in local.py for memcached.
-CACHES = {
-    'default': {
-        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
-        'LOCATION': 'translations'
-    }
-}
+CACHES = config(
```

Comment:
Same comment here. I kinda balk at JSON in env vars, but perhaps that's the way. Just kinds smells to me of a need for something else...